mozconfigs/smoosh-opt

```diff
@@ -0,0 +1,5 @@
+ac_add_options --enable-application=js
+ac_add_options --enable-optimize
```

Comment:
There is more to it, such as `--enable-release`.

Reply:
We're not using `--enable-release` in any spidermonkey jobs
https://searchfox.org/mozilla-central/source/js/src/devtools/automation/variants

is it important flag here?
if so, should we add it also to some variant?

Reply:
Yes, it is important because without it the Rust code is not optimized.

Reply:
okay, found that it's enabled on automation
https://searchfox.org/mozilla-central/source/build/moz.configure/toolchain.configure#8
Is there any other options that this mozconfig should contain?

Reply:
just noticed that opt contains `ac_add_options --enable-debug` by accident
it should be `ac_add_options --disable-debug`